main.py

```diff
@@ -265,7 +109,7 @@ def set_pixel_and_check_ratelimit(
         }
 
         response = requests.request(
-            "POST", url, headers=headers, data=payload, proxies=self.GetRandomProxy()
+            "POST", url, headers=headers, data=payload, proxies=proxy.GetRandomProxy()
```

Comment:
Does this method call not need to be updated?

Reply:
Oh wait